_includes/scripts.html

```diff
@@ -15,3 +15,12 @@
   {% assign _src = _script.src | default: _script %}
 <script src="{{ _src | replace: _async_marker, '' | relative_url }}"{% if _script.async or _src contains _async_marker %} async{% endif %}></script>
 {% endfor %}{% endfor %}
+
+{% if page.layout == 'search-results' %}
+<script>
+var JEKYLL_PAGES_API_SEARCH_UI_OPTIONS = {
+  inputElementId: 'search-field-small'
```

Comment:
The default uses ID `search-input` - if you change your markup to use that ID instead, you can exclude this `<script>` section.

Reply:
Thanks, I've made that update 👍